core/src/jsMain/kotlin/Peripheral.kt

```diff
@@ -250,4 +252,4 @@ public class JsPeripheral internal constructor(
     override fun toString(): String = "Peripheral(bluetoothDevice=${bluetoothDevice.string()})"
 }
 
-private fun ArrayBuffer.toByteArray(): ByteArray = Int8Array(this).unsafeCast<ByteArray>()
+internal fun ArrayBuffer.toByteArray(): ByteArray = Int8Array(this).unsafeCast<ByteArray>()
```

Comment:
"Borrowed" this extension for the advertisement data parsing. Should we move it somewhere else?

Reply:
Yes please. Perhaps a new file names `Buffers.kt` or `Bytes.kt`?